pages/standardisation.md

```diff
@@ -4,148 +4,12 @@ title: Standardisation
 permalink: standardisation
 ---
 
-You can read the [Solid Specification](https://solid.github.io/specification/) and the details of [how to participate in the standardisation process](https://github.com/solid/process) here.
+You can read the most up to date version of the [Solid Specification](https://solid.github.io/specification/).
 
-Please read the [Code of Conduct](#conduct) before doing so.
+Anyone can contribute to the Solid Specification by making suggestions by submitting [issues](https://github.com/solid/specification/issues) and [pull requests](https://github.com/solid/specification/pulls). Before contributing to the Solid Specification please read the [Code of Conduct](#conduct).
```

Comment:
Perhaps frame the specification work under the CG and invite them to join in order to contribute?